client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -22,14 +29,30 @@ import SettingsPaymentsOnSite from './payments-on-site';
 class SettingsPayments extends Component {
 
 	static propTypes = {
+		isSaving: PropTypes.bool,
 		site: PropTypes.shape( {
 			slug: PropTypes.string,
 		} ),
 		className: PropTypes.string,
 	};
 
+	onSave() {
+		const { translate, actions } = this.props;
+
+		const successAction = successNotice(
+			translate( 'Payment changes saved.' ),
+			{ duration: 4000 }
+		);
+
+		const failureAction = errorNotice(
+			translate( 'There was a problem saving the payment changes. Please try again.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'There was a problem saving your changes. Please try again.' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/store-payment-method-action-list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).